dirty_cat/test/benchmark_sprint.py

```diff
@@ -0,0 +1,162 @@
+import warnings
```

Comment:
This file should be moved to a new directory, called "benchmarks", at the root of the project. Also, we should find a better name for it :). And add a small docstring to the file to explain what it does.